plugin/trino-oracle/src/main/java/io/trino/plugin/oracle/OracleClient.java

```diff
@@ -242,6 +245,12 @@ public OracleClient(
                 .add(new RewriteStringComparison())
                 .build();
 
+        this.projectFunctionRewriter = new ProjectFunctionRewriter<>(
+                this.connectorExpressionRewriter,
+                ImmutableSet.<ProjectFunctionRule<JdbcExpression, ParameterizedExpression>>builder()
```

Comment:
How about?
` ImmutableSet.of(new RewriteCast(this::toWriteMapping)))`

Reply:
builder will reduce the difference when adding new rewrite class in future. So Ill keep the builder here.